weave

```diff
@@ -328,15 +300,82 @@ fi
 # main (remote and --local) - settings
 ######################################################################
 
+## Versions ##
+SCRIPT_VERSION="unreleased"
```

Comment:
You cannot move this after the first usage (which is at line 241 in this version)